findbugs-exclude.xml

```diff
@@ -0,0 +1,5 @@
+<FindBugsFilter>
+     <Match>
+       <Class name="~.+\.Messages" />
```

Comment:
For now, I have only discovered that findbugs marks the build as failed if the attribute in the properties file matches `[A-Z][^.]+` . You can try with:

```
Bonjour=Bonjour, {0}!
Hello=Hello, {0}!
```

Seems like the `.` in the property matters here.